items/src/main/scala/weco/api/items/services/ItemUpdateService.scala

```diff
@@ -1,62 +1,91 @@
 package weco.api.items.services
 
-import grizzled.slf4j.Logging
-import weco.api.stacks.services.SierraService
 import weco.catalogue.internal_model.identifiers.{
   IdState,
   IdentifierType,
   SourceIdentifier
 }
-import weco.catalogue.internal_model.locations.{
-  AccessCondition,
-  PhysicalLocation
-}
 import weco.catalogue.internal_model.work.{Item, Work, WorkState}
 
 import scala.concurrent.{ExecutionContext, Future}
 
+trait ItemUpdater {
+  val identifierType: IdentifierType
+  def updateItems(
+    items: Seq[Item[IdState.Identified]]
+  ): Future[Seq[Item[IdState.Identified]]]
+}
```

Comment:
Minor: move this to a new file?

Nice idea to put the `identifierType` on the ItemUpdater trait.